stl/inc/flat_set

```diff
@@ -274,14 +289,16 @@ public:
         _Insert_range<true>(_Ilist.begin(), _Ilist.end());
     }
 
-    _NODISCARD container_type extract() && {
+    _NODISCARD container_type extract() && noexcept(is_nothrow_move_constructible_v<_Container>) /* strengthened */ {
         // always clears the container (N4950 [flat.set.modifiers]/14 and [flat.multiset.modifiers]/10)
-        _Clear_scope_guard<_Base_flat_set> _Guard{this};
+        _Clear_guard<_Base_flat_set, is_nothrow_move_constructible_v<_Container>> _Guard{this};

```

Comment:
Hmm, it seems that a successful move-construction cannot guarantee the container being emptied, so this enhancement is invalid... Will fix it soon.

Reply:
fixed in the new pr.